lib/vye/dgib/authentication_token_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Vye
+  module DGIB
+    class AuthenticationTokenService
+      ALGORITHM_TYPE = 'RS256'
+      E = 'AQAB'
+      TYP = 'JWT'
+      KID = 'vye'
+      USE = 'sig'
+      SIGNING_KEY = Settings.dgi.vye.jwt.private_key_path
+      RSA_PRIVATE = OpenSSL::PKey::RSA.new(File.read(SIGNING_KEY)) if File.exist?(SIGNING_KEY)
+
+      def self.call
+        payload = {
+          exp: Time.now.to_i + (5 * 60), # JWT expiration time (5 minutes)
```

Comment:
Could you change this to `5.minutes.from_now.to_i`. It's more reliable and 'rails like'

Reply:
this change has been made